openapi-core/src/main/java/io/ballerina/openapi/core/generators/document/TypesDocCommentGenerator.java

```diff
@@ -128,7 +131,7 @@ public SyntaxTree updateSyntaxTreeWithDocComments() {
             }
         });
         NodeList<ModuleMemberDeclarationNode> typeMembers = AbstractNodeFactory.createNodeList(
```

Comment:
This represents the non-type member nodes right?

Reply:
no all the members. will rename to `updatedMembers`.

Reply:
updated